src/components/Card/Card.js

```diff
@@ -1,16 +1,37 @@
-import { Tile } from 'carbon-components-react';
-import React from 'react';
+import { Tile } from "carbon-components-react";
+import React, { useState } from "react";
 
-const Card = () => (
-    <Tile className='bx--tile--clickable card'>
-        <div className='card__header'>
-            Header
-        </div>
-        <hr/>
-        <div className='card__content'>
-            Content
-        </div>
+const Card = () => {
+  const uncheckedCardStyle = "bx--tile--clickable card";
+  const checkedCardStyle = "bx--tile--clickable card__checked";
+
+  const [isChecked, setIsChecked] = useState(false);
+
+  function checkBoxClickHandler() {
+    setIsChecked(!isChecked);
+  }
+
+  return (
+    <Tile
+      className={
+        isChecked
+          ? checkedCardStyle
+          : uncheckedCardStyle
+      }
+    >
```

Comment:
... or you can do it like this:
`className={"bx--tile--clickable " + ( isChecked ? "card__checked" : "card") }`